googleSchema.js

```diff
@@ -0,0 +1,55 @@
+// Philosophy FAQ
+// to add more philosophy questions just add them in this json as the first 2 and the script will do the rest
+var faqPhilosophy = {
```

Comment:
Don't use var, change anything that doesn't need to be mutated to `const`, and anything that *needs* to be mutated to `let`.

Reply:
Changed the main array to const and the other strings to let